src/libraries/System.Threading/tests/InterlockedTests.cs

```diff
@@ -603,6 +603,22 @@ public void InterlockedAddAndRead_Multithreaded_Int64()
             Assert.Equal(ThreadCount * IterationCount * Increment, Interlocked.Read(ref value));
         }
 
+        static Action mb;
+        static delegate* <void> pMb;
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
Does this really need to be under `IsThreadingSupported`? I would expect MemoryBarrier to be no-op in single threaded modes.